### PR TITLE
Suggested changes to anonymous user selection

### PR DESCRIPTION
**Pull request by marfavi**
moved location of setting to the Your Profile view
added loading indicator when toggling between on/off
text of setting changed to "Appear anonymous in leaderboard" (or "Appear anonymous" on iPhone 5-sized screens)
added description for setting group

### CODE REVIEW

File: lib/base/strings.dart [hunk 1]
```diff
@@ -197,7 +197,8 @@ abstract class Strings {
   static const change = 'Change';
   static const logOut = 'Log out';
   static const deleteAccount = 'Delete account';
-  static const showOnLeaderboard = 'Show me on the leaderboard';
+  static const appearAnonymous = 'Appear anonymous in leaderboard';
```
Comment (fremartini) [inline lib/base/strings.dart:200]: should be 'on' leaderboard

**Review by fremartini — CHANGES_REQUESTED** [review]

File: lib/widgets/pages/settings/your_profile_page.dart [hunk 2]
```diff
@@ -18,80 +19,85 @@ class YourProfilePage extends StatelessWidget {
...
+              valueWidget: const SettingDescription(),
+              onTap: () {},
+            ),
+            SettingListEntry(
+              name: deviceIsSmall(context)
+                  ? Strings.appearAnonymousSmall
+                  : Strings.appearAnonymous,
+              valueWidget: Switch(
+                value: user.privacyActivated,
+                onChanged: (privacyActived) async {
```
Comment (fremartini) [inline lib/widgets/pages/settings/your_profile_page.dart:91]: can you verify this value is the expected one? as in true when toggled on and false when off

**Comment by marfavi** [comment]
> image should this be centered?

No, see pictures from design spec
image
image
image

File: lib/widgets/pages/settings/your_profile_page.dart (hunk 2 above)
Comment (marfavi) [reply lib/widgets/pages/settings/your_profile_page.dart:91]: looks correct to me

File: lib/base/strings.dart (hunk 1 above)
Comment (marfavi) [reply lib/base/strings.dart]: fixed in new commit, also changed it in the settings group description

**Pull request merged**